charts/backstage/templates/backstage-deployment.yaml

```
@@ -36,6 +36,10 @@ spec:
       securityContext:
         {{- include "common.tplvalues.render" ( dict "value" .Values.backstage.podSecurityContext "context" $) | nindent 8 }}
       {{- end }}
+      {{- with .Values.backstage.nodeSelector }}
+      nodeSelector:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
```

```suggestion
      {{- if .Values.backstage.nodeSelector }}
      nodeSelector:
        {{- include "common.tplvalues.render" ( dict "value" .Values.backstage.nodeSelector "context" $) | nindent 8 }}
      {{- end }}
```

I think we want to allow value propagation and templating here. It can improve the user experience.